arangod/Transaction/Methods.h

```diff
@@ -414,6 +414,8 @@ class Methods {
   /// @brief build a VPack object with _id, _key and _rev and possibly
   /// oldRef (if given), the result is added to the builder in the
   /// argument as a single object.
+
+  //SHOULD THE OPTIONS BE CONST?
```

Comment:
which options?

Reply:
The OperationOptions that get passed into the following calls.

Reply:
They are already const... So where's is the problem?

Reply:
https://github.com/arangodb/arangodb/blob/7d5d8300702c07345d9f7f68a788495196ee35f1/arangod/Transaction/Methods.h#L428